linux_os/guide/services/ssh/ssh_server/sshd_disable_root_password_login/rule.yml

```diff
@@ -0,0 +1,35 @@
+documentation_complete: true
+
+title: 'Disable SSH root Login with a Password (Insecure)'
```

Comment:
Why to have `(insecure)` in the title?
You have rationale and warning that describe that describe the problem of password-based auth and the problem of root being able to log in directly respectively, so there is no need to indicate insecurity of either in the title.

Reply:
Sure there is. This option opens root account to remote attack and should be obvious in every way that using this rule is insecure

Reply:
How about `Disable SSH root Login with a Password (Still permit remote root logins)`?

Reply:
@jcpunk no... keep the insecure wording

Reply:
Insecure kept.